src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -59,6 +61,30 @@ export const mhvBaseUrl = () => {
   return `https://${mhvSubdomain}.myhealth.va.gov`;
```

Comment:
can the `mhvBaseUrl` function be deleted?

Reply:
yup, will do
- [ ] Remove `mhvBaseUrl`

Reply:
@ericbuckley actually, there's one file that actually still needs this function, I realized. 

`src/platform/utilities/environment/stagingDomains.js` uses it for literally a base url. The changes I made to `mhvUrl` make some additional assumptions, where it tacks on that `/mhv-portal-web` subroute. Given that, I'll keep this around for now